test/irmin-pack/test_lower.ml

```diff
@@ -294,6 +294,71 @@ module Store_tc = struct
     let* _ = Store.Gc.start_exn repo (Store.Commit.key b_commit) in
     let* _ = Store.Gc.finalise_exn ~wait:true repo in
     Store.Repo.close repo
+
+  let test_volume_data_locality () =
+    let root, lower_root = fresh_roots () in
+    let* repo = Store.Repo.v (config ~fresh:true ~lower_root root) in
+    let* main = Store.main repo in
+    let info () = Store.Info.v ~author:"test" Int64.zero in
+    [%log.debug "add c1"];
+    let* () = Store.set_exn ~info main [ "c1" ] "a" in
+    let* c1 = Store.Head.get main in
+    [%log.debug "GC c1"];
+    let* _ = Store.Gc.start_exn repo (Store.Commit.key c1) in
+    let* _ = Store.Gc.finalise_exn ~wait:true repo in
+    let () = Store.add_volume repo in
+    [%log.debug "add c2, c3, c4"];
+    let* () = Store.set_exn ~info main [ "c2" ] "b" in
+    (* let* _c2 = Store.Head.get main in *)
+    let* () = Store.set_exn ~info main [ "c3" ] "c" in
+    let* c3 = Store.Head.get main in
+    let* () = Store.set_exn ~info main [ "c4" ] "d" in
+    (* let* c4 = Store.Head.get main in *)
```

Comment:
Perhaps we can remove this comment? (same for `_c2` above?)

Reply:
:see_no_evil: yes! Thanks for spotting.